src/meetingsession/MeetingSessionStatusCode.ts

```diff
@@ -24,20 +24,32 @@ export enum MeetingSessionStatusCode {
   AudioDisconnectAudio = 3,
 
   /**
-   * Authentication was rejected. The client is not allowed on this call.
+   * Authentication was rejected. The client is not allowed on this meeting.
    */
   AudioAuthenticationRejected = 4,
 
   /**
-   * The client can not join because the call is at capacity.
+   * The client can not join because the meeting is at capacity.
    */
   AudioCallAtCapacity = 5,
 
   /**
-   * The call was ended.
+   * Deprecated. The meeting has ended. This is a legacy alias for MeetingEnded and will
+   * be removed in v2.0.0.
    */
   AudioCallEnded = 6,
 
+  /**
+   * Deprecated. The meeting has ended. This is a legacy alias for MeetingEnded and will
+   * be removed in v2.0.0.
+   */
+  TURNMeetingEnded = 6,
+
+  /**
+   * The meeting has ended.
+   */
+  MeetingEnded = 6,
```

Comment:
We changed the `TURNMeetingEnded = 24` to `TURNMeetingEnded = 6` here. Isn’t that backward incompatible?

Reply:
It is ok to change the underlying internal value since these numbers are only used to differentiate the enum values. As long as the client application uses the names, the behavior is the same. This change has been verified in the meeting demo.